### DISCUSSION TITLE
How long is login blocked after "Error: control/login | auth: blocked for $MACAddress | 429"

### DISCUSSION THREAD

**Discussion by technicaladvantage** [AdguardTeam/AdGuardHome#6805] [Q&A]
`"Error: control/login | auth: blocked for $MACAddress | 429"`

How long is login blocked after this?

Is there a way to kill the block via cli or the gui?

**Comment by ghost** [comment]
15mins by default, it tells you in the error message:
`Error: control/login | auth: blocked for 14m59.512717248s | 429`
